_data/HarshitaArya/HarshitaArya.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Harshita Arya
+institution: Kalinga Institute of Industrial Technology Deemed to be University
```

Comment:
The name of the institution must **not** be longer than 58 characters, yours is **66** characters.